testassets/InteropTestsClient/InteropTestsClient.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks Condition="'$(LatestFramework)'!='true'">net6.0;net5.0;net472</TargetFrameworks>
-    <TargetFrameworks Condition="'$(LatestFramework)'=='true'">net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(LatestFramework)'!='true'">net7.0;net6.0;net5.0;net472</TargetFrameworks>
+    <TargetFrameworks Condition="'$(LatestFramework)'=='true'">net7.0</TargetFrameworks>
```

Comment:
I think updating this will break the interop tests on grpc/grpc? We'd need multiple separate changes to upgrade grpc/grpc's interop for grpc-dotnet to net7.

Reply:
Maybe. This is already passing on interop tests CI for this PR.